### DISCUSSION TITLE
Hoppscotch Localhost Post Failure

### DISCUSSION THREAD

**Discussion by 14jchan** [hoppscotch/hoppscotch#3388] [Q&A] [answered]
I have a strange situation with Hoppscotch. I am trying to make a POST request (GET requests work just fine) to my localhost:8081 (yes, the port is correct).

```
    curl --request POST \
      --url http://localhost:8081/
```

The same error would happen in my cmd.

I get a 404, even though it is clearly defined. Before postman took away the scratchpad, commands like this would work just fine. See some images here. Any ideas how to fix this in hoppscotch?

I have attached an image of the sample springboot method, as well as the cmd and hoppscotch. I am currently using an chrome extension, so localhost calls do work for GET.

What else is strange is that POST requests onto actual URLs seem to work....

Here are some screenshots: 

1. https://i.stack.imgur.com/nNBbE.png
2. https://i.stack.imgur.com/RAd1V.png
3. https://i.stack.imgur.com/uILrT.png

**Comment by liyasthomas** [comment, marked as answer]
Thanks for your patience and valuable feedback. This issue has been resolved in the latest release.
